recipes/jupyterlab-lsp/modify_conftest.py

```diff
@@ -0,0 +1,20 @@
+import re
+import sys
+from pathlib import Path
+
+from jupyter_lsp import tests
+
+
+if __name__ == "__main__":
+    # Reduce the number of known language servers
+    conftest = Path(tests.__file__).parent / "conftest.py"
+    conf = conftest.read_text()
+
+    servers = ",".join(map(lambda s: '"{:s}"'.format(s), sys.argv[1:]))
+    new_conf = re.sub(
+        r"KNOWN_SERVERS = \[(.*?)\]", 
```

Comment:
I think this may be running afoul of there being both `KNOWN_SERVERS` and `UNKNOWN_SERVERS` in that file, which this is also hitting. Of course, it's very unfair to COBOL, as there most certainly _is_ a COBOL language server!

Reply:
I remove the dirty hack to use a cleaner strategy using `-k` pytest option. The drawback is the need to list all unit tests making use of the known servers fixture (see Jinja variable `server_tests` at the top of the recipe).